### PR TITLE
Log errors

### PR DESCRIPTION
**Pull request by proteusvacuum**
If I've understood what this does https://github.com/dimagi/commcare-hq/blob/master/corehq/util/log.py#L57-L66 correctly, this should also send a stacktrace by email if it is available, right?

Follow up from: https://github.com/dimagi/commcare-hq/pull/10543

@czue

### CODE REVIEW

**Comment by czue** [comment]
I don't this will email. You can use `notify_error` or `notify_exception` to email.

File: corehq/apps/app_manager/views/modules.py [hunk 1]
```diff
@@ -631,9 +633,15 @@ def edit_report_module(request, domain, app_id, module_id):
...
     module.name = params['name']
+
     try:
         module.report_configs = [ReportAppConfig.wrap(spec) for spec in params['reports']]
-    except Exception:
+    except Exception as e:
+        notify_exception(
+            request,
+            message="Something went wrong while editing report modules",
+            details={'domain': domain, 'app_id': app_id,}
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/modules.py]: E231 missing whitespace after ','

File: corehq/apps/app_manager/views/modules.py [hunk 2]
```diff
@@ -644,7 +652,12 @@ def edit_report_module(request, domain, app_id, module_id):
 
     try:
         app.save()
-    except Exception:
+    except Exception as e:
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/modules.py]: F841 local variable 'e' is assigned to but never used

File: corehq/apps/app_manager/views/modules.py [hunk 3]
```diff
@@ -644,7 +652,12 @@ def edit_report_module(request, domain, app_id, module_id):
...
+        notify_exception(
+            request,
+            message="Something went wrong while saving app {} while editing report modules".format(app_id),
+            details={'domain': domain, 'app_id': app_id,}
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/modules.py]: E231 missing whitespace after ','

**Comment by czue** [comment]
:love_letter:

**Pull request merged**